package.json

```diff
@@ -23,8 +26,8 @@
   },
   "homepage": "https://github.com/bholloway/adjust-sourcemap-loader",
   "dependencies": {
-    "camelcase": "5.0.0",
-    "loader-utils": "1.2.3",
+    "camel-case": "^4.1.1",
```

Comment:
I can see why to update `loader-utils`. But what was the reason for changing `camelcase` to `camel-case`?

FWIW the `camelcase` has zero dependencies and is a more popular library.

Reply:
This is why I bumped node support
https://github.com/sindresorhus/camelcase/blob/master/package.json#L13-L15

Reply:
Yes sorry only just noticed your reply above.

If your aim is to update dependencies and use carat then I would accept `"camelcase": "^5.3.1"`.

From my point of view I prefer not to invest time in QA of large changes, since there is much to do on `resolve-url-loader`.

Reply:
I took a look at camelcase usage. And I don't see a value in it. For some reason format and codec name has different case. Is this necessary? Such breaking change would not be that critical and I would help to point people to changelog.

Reply:
Check please https://github.com/bholloway/adjust-sourcemap-loader/pull/15